Source/UnrealGT/Private/Generators/Image/GTSceneCaptureComponent2D.cpp

```diff
@@ -8,6 +8,8 @@
 #include <Engine/TextureRenderTarget2D.h>
 #include <Materials/MaterialInstanceDynamic.h>
 
+const FColor KBlack = FColor(0, 0, 0, 255);
```

Comment:
AFAIK, unreal already has a constant for black `FColor::Black`. I think that should be used rather than defining our own.

Reply:
Thanks! I didn't know about that. Still learning unreal.